### DISCUSSION TITLE
Images and raw content don't work unless VPN is on

### DISCUSSION THREAD

**Discussion by AptiviCEO** [community/community#23622] [Projects and Issues]
<p>Hello, guys!</p>
<p>Ever since the middle of October, the moment when I moved to a new house, the images on all the repositories don’t show up anymore. They try to load, but they look like they’re on the infinite loop. If I stopped the page, the images appear broken; they show the broken image icon.</p>
<p>This issue only affects the real images put anywhere that accepts them; the icons and the entire GitHub load fine. This also affected the profile picture.</p>
<p>Moreover, if I opened any file (text or binary) and told GitHub to load it in raw mode, it goes into an infinite loop trying to load until it says that it has timed out.</p>
<p>As a side-effect, I can’t seem to go back to a previous page using the back button, but if I refreshed the page, it goes back correctly and I can continue with my work. Also, I can’t even download anything from the Releases page of any repository.</p>
<p>However, if I launched my VPN program and connected, everything works fine; images load fine, raw content open fine.</p>
<p>I made sure that my browser is up to date on multiple devices as this issue affected all devices that tried to access GitHub, regardless if it was on my tablet, my phone, or my computer.</p>
<p>I tried to change the DNS to Cloudflare, but it didn’t fix it.</p>
<p>The screenshots are attached showing the problem in action. I know that I have to turn on the VPN to access GitHub fully, but I need a more permanent fix so I can access GitHub without VPN.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/1/316bb09f92196cb7de04b023a0c0673c5ea8c83c.png" data-download-href="/uploads/short-url/73c7tRPPUPkv30Dt7uQEk883WNe.png?dl=1" title="Screenshot_2021-12-25-13-50-45" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15963131/181078287-54d33808-c4dc-4dfd-8373-ae96f5b632db.png" alt="Screenshot_2021-12-25-13-50-45" data-base62-sha1="73c7tRPPUPkv30Dt7uQEk883WNe" width="292" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/1/316bb09f92196cb7de04b023a0c0673c5ea8c83c_2_292x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/1/316bb09f92196cb7de04b023a0c0673c5ea8c83c_2_438x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/1/316bb09f92196cb7de04b023a0c0673c5ea8c83c_2_584x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/1/316bb09f92196cb7de04b023a0c0673c5ea8c83c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_2021-12-25-13-50-45</span><span class="informations">600×1024 109 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/9/4945a791e95d9c662556f1abbc0eb1d54adcabe5.png" data-download-href="/uploads/short-url/asc5yjjfBtxLeriAWuyfff7t1Fb.png?dl=1" title="Screenshot_2021-12-25-13-50-36" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15963131/181078288-5c22c609-753b-4197-97e7-950240ba473c.png" alt="Screenshot_2021-12-25-13-50-36" data-base62-sha1="asc5yjjfBtxLeriAWuyfff7t1Fb" width="292" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/9/4945a791e95d9c662556f1abbc0eb1d54adcabe5_2_292x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/9/4945a791e95d9c662556f1abbc0eb1d54adcabe5_2_438x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/9/4945a791e95d9c662556f1abbc0eb1d54adcabe5_2_584x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/9/4945a791e95d9c662556f1abbc0eb1d54adcabe5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_2021-12-25-13-50-36</span><span class="informations">600×1024 97.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/5/055e223b6a675cffc6ca2dbec87ac034e3edce26.png" data-download-href="/uploads/short-url/Lu4d47hnTos86dpo3Y9dWcaSuq.png?dl=1" title="Screenshot_2021-12-25-13-50-01" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15963131/181078290-4fb27d9d-12d6-4a44-8537-9854af27dd05.png" alt="Screenshot_2021-12-25-13-50-01" data-base62-sha1="Lu4d47hnTos86dpo3Y9dWcaSuq" width="292" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/055e223b6a675cffc6ca2dbec87ac034e3edce26_2_292x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/5/055e223b6a675cffc6ca2dbec87ac034e3edce26_2_438x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/5/055e223b6a675cffc6ca2dbec87ac034e3edce26_2_584x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/055e223b6a675cffc6ca2dbec87ac034e3edce26_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_2021-12-25-13-50-01</span><span class="informations">600×1024 33 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is it an issue from GitHub’s end?</p>

**Comment by jfullstackdev** [comment]
<p>it’s working at my end, if that is from GitHub, I will not be seeing this</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/b/abac57947dae5b07224855fc1c764045fb92fb22.png" data-download-href="/uploads/short-url/ouGT6dk9Rgg816POkT9bIoKJpUS.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078294-c8e742b3-df3d-4bcf-9a09-9e6119353c21.png" alt="image" data-base62-sha1="ouGT6dk9Rgg816POkT9bIoKJpUS" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/b/abac57947dae5b07224855fc1c764045fb92fb22_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/b/abac57947dae5b07224855fc1c764045fb92fb22_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/b/abac57947dae5b07224855fc1c764045fb92fb22_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/b/abac57947dae5b07224855fc1c764045fb92fb22_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 370 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/2/22259e54d2f2a625547aae37d6a26b9102386233.png" data-download-href="/uploads/short-url/4S4Pxdcpj4qoB1ih6YCfoXWU6S7.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078295-cf0287ff-89f2-4fd7-a612-e623a92ec910.png" alt="image" data-base62-sha1="4S4Pxdcpj4qoB1ih6YCfoXWU6S7" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/2/22259e54d2f2a625547aae37d6a26b9102386233_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/2/22259e54d2f2a625547aae37d6a26b9102386233_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/2/22259e54d2f2a625547aae37d6a26b9102386233_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/2/22259e54d2f2a625547aae37d6a26b9102386233_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 462 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>Your description sounds a lot like an issue with routing between your ISP and GitHub, or possibly a misconfigured firewall that eats the packets. The page loading with VPN means that GitHub’s servers themselves are working, but you can’t reach them without the detour via the VPN operator.</p>
<p>I’d check my router for the firewall possibility (though it seems unlikely), and if that doesn’t help contact the ISP.</p>

**Comment by AptiviCEO** [comment]
<p>Thanks for help!</p>
<p>I see that it’s most likely coming from the ISP. I will try and get in touch with them once I see fit.</p>
<p>Again, thank you so much!</p>
